<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myfieinfo</class>
 <widget class="QWidget" name="myfieinfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>82</y>
     <width>340</width>
     <height>177</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="Last_Lay">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Adobe 宋体 Std L</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>文件名称：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Adobe 宋体 Std L</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>文件大小：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Adobe 宋体 Std L</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>文件创建时间：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Adobe 宋体 Std L</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>最近被访问时间：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Adobe 宋体 Std L</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>最近被修改时间：</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit_filename">
        <property name="font">
         <font>
          <family>Adobe 宋体 Std L</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_filesize">
        <property name="font">
         <font>
          <family>Adobe 宋体 Std L</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_createname">
        <property name="font">
         <font>
          <family>Adobe 宋体 Std L</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_lastvisited">
        <property name="font">
         <font>
          <family>Adobe 宋体 Std L</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_lastnodify">
        <property name="font">
         <font>
          <family>Adobe 宋体 Std L</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="checkBox_isdir">
        <property name="text">
         <string>文件夹</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_isfile">
        <property name="text">
         <string>文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_canread">
        <property name="text">
         <string>可读</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_canwrite">
        <property name="text">
         <string>可写</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_isexe">
        <property name="text">
         <string>可执行</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_isHidden">
        <property name="text">
         <string>隐藏</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
